File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/UpdateRestoreHarvestedPlantsRequest.java
package io.github.thunkier.thunkmetrc.wrapper.models;import java.util.List;
import com.fasterxml.jackson.annotation.JsonProperty;public class UpdateRestoreHarvestedPlantsRequest {
    @JsonProperty("HarvestId")
    public Integer harvestId;
    @JsonProperty("PlantTags")
    public List<Object> plantTags;
}

File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/UpdateRestoreHarvestedPlantsRequest.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class UpdateRestoreHarvestedPlantsRequest(
    @JsonProperty("HarvestId")
    val harvestId: Int? = null,
    @JsonProperty("PlantTags")
    val plantTags: List<Any>? = null
)
